conf/development-websocket.ini

```diff
@@ -1,6 +1,9 @@
 [app:main]
 use: call:h.websocket:create_app
 
+# Elasticsearch configuration
+es.url: http://localhost:9201
```

Comment:
Honestly just an ignorant question: does `development-websocket` need access to ES? I don't know what it's used for, hence I'm asking!

Reply:
It does need it (horrifyingly)

Reply:
I can't think of a logical reason why the websocket server would need search functionality nowadays and couldn't see any references to `h.search` in `h.websocket`, `h.realtime` or `h.streamer` other than the `config.include` line that Hannah referenced. The actual realtime update functionality appears to work without it too.

I think this may simply be a vestige from back when Elasticsearch was the primary data store for annotation data.

The work I did last week established that it would be worthwhile to move the websocket server into a separate container so it can be deployed and scaled separately. Explicitly removing the dependence on Elasticsearch might make this a little easier later.

Reply:
However I'm fine with not trying to remove that dependency in this PR. We can do that separately.

Reply:
Follow up PR - https://github.com/hypothesis/h/pull/5062

Reply:
Huh, nice